tests/e2e/cucumber.hooks.js

```diff
@@ -0,0 +1,31 @@
+const { Before, BeforeAll, AfterAll, After, setDefaultTimeout } = require('@cucumber/cucumber')
+const { firefox } = require('playwright')
+const { deleteFiles } = require('./utils/helpers')
+// default timeout (in milliseconds)
+setDefaultTimeout(100000)
+
+// launch the browser
+BeforeAll(async function () {
+  global.browser = await firefox.launch({
+    headless: false,
+    slowMo: 0
```

Comment:
the values should be taken from config
```suggestion
    headless: <from-config>,
    slowMo: <from-config>
```